README.md

```diff
@@ -99,6 +99,30 @@ NAME      AGE
 example   15s
 ```
 
+After a couple of minutes, all cluster members should become ready.
+
+```
+$ kubectl get zookeepercluster
+
+NAME      REPLICAS   READY REPLICAS   VERSION   DESIRED VERSION   INTERNAL ENDPOINT    EXTERNAL ENDPOINT   AGE
+example   3          3                latest                      10.100.200.18:2181   N/A                 94s
```

Comment:
after the latest merge, the version will no longer be `latest`

Reply:
I have changed that.